app/models/course.rb

```diff
@@ -619,6 +620,34 @@ def validate_subject_consistency
     end
   end
 
+  def validate_custom_age_range
+    # This regex checks that a 1 or 2 digit number is selected for the 'to' and 'from' values.
+    # It also checks that the age range is correctly formatted with _to_ in between these values
+    valid_regex_pattern = /^(\d{1,2})_to_(\d{1,2})$/
```

Comment:
I'm not a great fan of regexs that require a comment :grimacing: if there isn't a simpler comment at the very least i think this should be extracted out into a method something along the lines of `get_age_range` which returns a hash something like `{from: a, to: b}`

Otherwise when I read this method i'm immediately greeted by a comment and a regex that not everyone will understand - and it takes away from the actual validation logic

Reply:
I think it's a pretty straightforward regex. I just have not really used them much, so I mainly left it for people who don't know how they work (we have new juniors starting soon). I really think a regex is correct here due to the fact that our age range in years needs to follow a very specific format. 

Hopefully, I've addressed the rest of your comment though.